puppet/manifests/activemq-development-new.pp

```diff
@@ -0,0 +1,6 @@
+node 'dev-box' {
```

Comment:
Indeed, I have tested the software with the new activemq 5.14 version, and all tests pass.
Deleted the .new files.